components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -166,12 +191,24 @@ end
 ---@param widgets Widget[]
 ---@return Widget[]
 function CustomInjector:parse(id, widgets)
-	local args = self.caller.args
+	local caller = self.caller
+	local args = caller.args
+	local data = caller.data
 
 	if id == 'gamesettings' then
 		return {
-			Cell{name = 'Game Version', content = {self.caller:_getGameVersion(args)}},
-			Cell{name = 'Server', content = {self.caller:_getServer(args)}}
+			Cell{name = 'Game Version', content = {caller:_getGameVersion(args)}},
+			Cell{name = 'Server', content = {caller:_getServer(args)}}
+		}
+	elseif id == 'dates' and data.startTime.display then
+		local startTime = Countdown._create{date = data.startTime.display, rawdatetime = true}
+
+		if data.startDate == data.endDate then
+			return {Cell{name = 'Date', content = {startTime}}}
+		end
+		return {
+			Cell{name = 'Start Date', content = {startTime}},
+			Cell{name = 'End Date', content = {args.edate}},
```

Comment:
The cell names don't work that well with times as well. Should be `Start Time` if time is supplied. And for L207, should also be `Start Time` rather than `Date`